lib/media/drm_engine.js

```diff
@@ -1710,6 +1710,15 @@ shaka.media.DrmEngine = class {
 
     const testSystem = async (keySystem) => {
       try {
+        // Our Polyfill will reject anything apart com.apple.fps key systems.
+        // It seems the Safari modern EME API will allow to request a
+        // MediaKeySystemAccess for the ClearKey CDM, create and update a key
+        // session but playback will never start
+        if (keySystem === 'org.w3.clearkey' &&
```

Comment:
I think it would be good to add the following link: https://bugs.webkit.org/show_bug.cgi?id=231006